Client/Frontend/Popup/AlertPopupView.swift

```diff
@@ -56,32 +56,54 @@ class AlertPopupView: PopupView {
     }
     
     func updateSubviews() {
+        titleLabel.adjustsFontSizeToFitWidth = false
+        messageLabel.adjustsFontSizeToFitWidth = false
+
+        _updateSubviews()
+        
+        let paddingHeight: CGFloat = padding * 3.0
+        let externalContentHeight: CGFloat = dialogButtons.count == 0 ? paddingHeight
+            : kPopupDialogButtonHeight + paddingHeight
+        
+        if containerView.frame.height + externalContentHeight > UIScreen.main.bounds.height {
+            let desiredHeight: CGFloat = UIScreen.main.bounds.height - externalContentHeight
```

Comment:
This logic is actually functionally equivalent to the `if` conditional logic right above it. Probably cleaner to create this constant outside the `if` and just use it in the boolean logic:

```
let desiredHeight: CGFloat = UIScreen.main.bounds.height - externalContentHeight
if containerView.frame.height > desiredHeight {
    // use `desiredHeight`
}
```